apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -564,4 +566,23 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
     public getBalanceDue(): number {
         return parseFloat(Number(this.adjustPayment.grandTotal + this.adjustPayment.tcsTotal - this.adjustPayment.totalAdjustedAmount - this.depositAmount - this.adjustPayment.tdsTotal).toFixed(2));
     }
+
+    /**
+     * To check form validation
+     *
+     * @memberof AdvanceReceiptAdjustmentComponent
+     */
+    public checkValidations() {
+        if (this.adjustVoucherForm && this.adjustVoucherForm.adjustments && this.adjustVoucherForm.adjustments.length > 0) {
+            this.adjustVoucherForm.adjustments.forEach((item) => {
+                if (!item.voucherNumber && item.dueAmount.amountForAccount) {
```

Comment:
This can be simplified as:
`if (item.voucherNumber || item.dueAmount.amountForAccount) {
   this.isInvalidForm = true;
} else {
   this.isInvalidForm = false;
}
`
Add nullity check `item.dueAmount`

Reply:
No need to add because we have prepare object on init method